charts/prometheus-node-exporter/templates/_helpers.tpl

```diff
@@ -200,3 +200,36 @@ labelValueLengthLimit: {{ . }}
 {{- end }}
 {{- end }}
 {{- end }}
+
+{{/*
+The default node affinity to exclude Fargate / Virtual nodes
```

Comment:
```suggestion
The default node affinity to exclude
- AWS Fargate
- Azure Virtual nodes
```

Reply:
Thanks for the suggestion, adapted your changes